### DISCUSSION TITLE
Keyboard shortcut settings on ZSA Ergodox-ez

### DISCUSSION THREAD

**Discussion by mutalis** [gilesknap/notes#3] [Show and tell]
Hey Giles

Thanks for sharing your keyboard shortcut configuration settings.

I'm using the ZSA Ergodox-ez keyboard and almost all the Vscode shortcuts worked for me. I found that one possible reason for such an issue was another App listening for the same shortcut.

Reference: https://github.com/microsoft/vscode/issues/210106#issuecomment-2060039608

In the end, I had to do some shortcuts changes to get all working.

My keyboard layout:
https://configure.zsa.io/ergodox-ez/layouts/Ql50m/latest/0

Using: Mac OSX Sonoma

Regards,
Rodolfo.

**Comment by gilesknap** [comment]
So that is a little annoying, the reason for using hyperkey is that it is unlikely to clash with other bindings. Which app is already using hyperkey? They must expect their (non-programmable keyboard) users to contort their fingers!
